### PR TITLE
Display app version as well as contracts version

### LINKED ISSUE #2081: Display both app version and contracts version in the user settings modal
### Problem

in `TabWrapper.vue`, we currently only show the contracts version:

```js
      subNav: [{
        title: `${L('Version')} ${contractsVersion}`
      }, {
        title: L('Acknowledgements'),
        url: 'acknowledgements',
        component: 'Acknowledgements',
        index: 11 // NOTE: index should not be duplicated with the link of tabNav
      }]
```

Screenshot 2024-06-17 at 10 34 38 AM

This is an issue because there are actually two versions that we keep track of: the contracts version and the App Version.

Any time we update the frontend code (but not the contracts) we bump the App Version (but not the contracts). And any time we bump the contracts, we bump both the App Version and the Contracts Version.

### Solution

Display both versions. Make sure it looks good. You can test two different approaches: the versions as one string on one line, or as two separate lines.

E.g.:

```
Version: App (0.5.1) Contracts (0.5.0)
```

Vs:

```
App Version: 0.5.1
Contracts Version: 0.5.0
```

Pick whichever looks best in the UI with the limited space available.

### PR DESCRIPTION
**Pull request by Silver-IT**
### Summary of Changes

- Closes #2081

### CODE REVIEW

File: frontend/views/components/tabs/TabWrapper.vue [hunk 1]
```diff
@@ -53,14 +54,21 @@ export default ({
...
     return {
       activeTab: 0,
       activeComponent: null,
       title: '',
       transitionName: '',
       open: true,
       subNav: [{
-        title: `${L('Version')} ${contractsVersion}`
+        html: L('{span_}App Version: {appVersion}{_span}{br_}{span_}Contracts Version: {contractsVersion}{_span}', {
+          ...LTags('span'),
```
Comment (taoeffect) [inline frontend/views/components/tabs/TabWrapper.vue]: Is there a reason for this `span` tag?

I tried changing this to: `App Version: {appVersion}{br_}Contracts Version: {contractsVersion}`

And it worked just fine:

Screenshot 2024-06-19 at 10 31 53 AM

**Review by taoeffect — CHANGES_REQUESTED** [review]
Nicely done! One minor change request for simplifying the string for translators:

**Review by Silver-IT — COMMENTED** [review]
Review ready!

**Review by taoeffect — APPROVED** [review]
🫡

**Pull request merged**